docs/for-developers.md

```diff
@@ -66,6 +66,11 @@ To run the dev demo where features under development are usually being tested:
 bun run vs.example
 ```
 
+### Docker
```

Comment:
Uh oh. My editor automatically removed trailing whitespace. I believe there is an option when viewing a PR to hide that. If you want me to revert and try again, please let me know.